src/CalcManager/Ratpack/conv.cpp

```diff
@@ -944,22 +1016,22 @@ ULONGLONG rattoUlonglong( _In_ PRAT prat, uint32_t radix, int32_t precision)
 //
 //    ARGUMENTS: number input and base of that number.
 //
-//    RETURN: long
+//    RETURN: int32_t
 //
-//    DESCRIPTION: returns the long representation of the
+//    DESCRIPTION: returns the int32_t representation of the
 //    number input.  Assumes that the number is really in the
 //    base   claimed.
 //
 //-----------------------------------------------------------------------------
-long numtolong( _In_ PNUMBER pnum, uint32_t radix )
+int32_t numtolong( _In_ PNUMBER pnum, uint32_t radix )
```

Comment:
should we be updating names in this file?

Reply:
There's actually a few throughout the file that have specific types as part of their name that are now incorrect after this change.